scripts/dts/new_extract.py

```diff
@@ -18,6 +18,45 @@ def str2ident(s):
             .replace('/', '_') \
             .upper()
 
+def handle_prop(define, prop_name, prop, out, dev):
+
+    foo = False
+
+    if prop is None:
+        return
+
+    if type(prop) is bool:
```

Comment:
`isinstance(prop, bool)` is more common.

Reply:
changed